doc/wiki/feature-flags.md

```diff
@@ -23,7 +23,6 @@ to facility the launch of a new feature, and may reference the Simple team's int
 | imo_messaging | Yes | When enabled, patient reminder messages will be attempted via Imo first, before falling back to SMS. |
 | notifications | Yes | When enabled, Simple Server will send patient reminder messages. |
 | organization_reports | Yes | When enabled, the Simple Dashboard will allow users to view a top-level report for the whole organization. |
-| reporting_schema_v2 | Yes | Feature flag to launch [Reporting Pipeline](https://app.shortcut.com/simpledotorg/epic/2422/reporting-pipeline-etl) |
```

Comment:
This flag should no longer exist anywhere, as it is unnecessary now and full launched.